content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,656 @@
+aggregate and import data into Prometheus.
+
+While you can use Prometheus to monitor a variety of infrastructure and
+application metrics, one of its most popular use cases is to monitor Kubernetes.
+This is the aspect of Prometheus monitoring that we will focus on in this
+article.
+
+## Prometheus metrics with OpenTelemetry
+
+In this section, you'll learn about a couple of OTel Collector components that
+demonstrate the interoperability between OTel and Prometheus.
+
+First, let’s do a quick refresher on the [Collector](/docs/collector/) – it’s an
+OTel component that can be used to collect telemetry from multiple sources and
+export data to multiple destinations. The Collector also handles telemetry
+processing, such as modifying data attributes and scrubbing personally
+identifiable information. For example, you can use Prometheus SDKs to generate
+metrics, ingest them with the Collector, do some processing (if desired) and
+then forward them to your chosen backend.
+
+![Diagram showing the different components of the OTel Collector](OTel-collector-refresher.png)
+
+The
+[Prometheus receiver](/docs/kubernetes/collector/components/#prometheus-receiver)
+allows you to collect metrics from any software that exposes Prometheus metrics.
+It serves as a drop-in replacement for Prometheus to scrape your services, and
+supports the
+[full set](https://github.com/prometheus/prometheus/blob/v2.28.1/docs/configuration/configuration.md#scrape_config)
+of configurations in `scrape_config`.
+
+If you are interested in
+[exemplars](/docs/specs/otel/metrics/data-model/#exemplars), which is a recorded
+value that associates OTel context with a metric event, you can also use the
+Prometheus receiver to ingest them in the Prometheus format and convert it to
+OTLP format. This enables you to correlate traces with metrics.
+
+Something to consider with this component is that it is under active
+development; as such, it has several
+[limitations](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/README.md#%EF%B8%8F-warning),
+including that it’s a stateful component. Additionally, it is recommended to not
+use this component when multiple replicas of the Collector are run, because in
+this state:
+
+- The Collector is unable to auto-scale the scraping
+- If the replicas are running with the same config, it will scrape the targets
+  multiple times
+- You will need to configure each replica with a different scraping config if
+  you want to manually shard the scraping
+
+For exporting metrics from the OTel Collector to Prometheus, you have two
+options: the
+[Prometheus exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusexporter#prometheus-exporter),
+and the
+[Prometheus Remote Write exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusremotewriteexporter/README.md).
```

Comment:
Prometheus also supports OTLP natively now, this is something that should be mentioned, see https://prometheus.io/blog/2024/03/14/commitment-to-opentelemetry/

Reply:
I see now that you call this out in the conclusion, you might mention this hear already as well since otherwise people will not read to the bottom of the page and do not learn that.

Reply:
Sounds good! I added it to this section.